lib/Zonemaster/Backend/Config.pm

```diff
@@ -37,6 +37,20 @@ sub load_config {
     $self->{cfg} = Config::IniFiles->new( -file => $path );
     die "UNABLE TO LOAD $path ERRORS:[".join('; ', @Config::IniFiles::errors)."] \n" unless ( $self->{cfg} );
     bless( $self, $class );
+
+    if ( defined $self->{cfg}->val( 'DB', 'database_host' ) ) {
+        $log->warning( "Use of deprecated config property DB.database_host. Use MYSQL.host or POSTGRESQL.host instead." );
+    }
+    if ( defined $self->{cfg}->val( 'DB', 'user' ) ) {
+        $log->warning( "Use of deprecated config property DB.user. Use MYSQL.user or POSTGRESQL.user instead." );
+    }
+    if ( defined $self->{cfg}->val( 'DB', 'password' ) ) {
+        $log->warning( "Use of deprecated config property DB.password. Use MYSQL.password or POSTGRESQL.password instead." );
+    }
+    if ( defined $self->{cfg}->val( 'DB', 'database_name' ) ) {
+        $log->warning( "Use of deprecated config property DB.database_name. Use MYSQL.database, POSTGRESQL.database or SQLITE.file instead." );
+    }
```

Comment:
PR #717 must be merged before this is merged or else there is a risk of filling up the log files.

Reply:
I included this comment in the PR description.